Snippets/list-chars-in-font.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+from itertools import chain
+import sys
+
+from fontTools.ttLib import TTFont
+from fontTools.unicode import Unicode
+
+ttf = TTFont(sys.argv[1], 0, verbose=0, allowVID=0,
+                ignoreDecompileErrors=True,
+                fontNumber=-1)
+
+chars = chain.from_iterable([y + (Unicode[y[0]],) for y in x.cmap.items()] for x in ttf["cmap"].tables)
+print(list(chars))
```

Comment:
you could prettify the output by formatting it like a table, and maybe using hex digits for character codes instead of int
```
0x0021 exclam EXCLAMATION MARK
```